packages/vkui/src/components/CustomSelectOption/CustomSelectOption.tsx

```diff
@@ -106,7 +107,10 @@ export const CustomSelectOption = ({
       <div className={styles['CustomSelectOption__main']}>
         <div className={styles['CustomSelectOption__children']}>{children}</div>
         {hasReactNode(description) && (
-          <Footnote className={styles['CustomSelectOption__description']}>{description}</Footnote>
+          <>
+            <VisuallyHidden>&nbsp;</VisuallyHidden>
+            <Footnote className={styles['CustomSelectOption__description']}>{description}</Footnote>
```

Comment:
```suggestion
            <Footnote className={styles['CustomSelectOption__description']}><VisuallyHidden>&nbsp;</VisuallyHidden>{description}</Footnote>
```

а если так? чтобы не создавать фрагмент